uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -2065,4 +2070,59 @@ public function testLeaveRequestIsDeletedValueCanNotBeSetWhenCreatingALeaveReque
     $leaveRequestRecord = LeaveRequest::findById($leaveRequest->id);
     $this->assertEquals(0, $leaveRequestRecord->is_deleted);
   }
+
+  public function testEmailGetsSentWhenLeaveRequestIsCreated() {
+    $manager1 = ContactFabricator::fabricateWithEmail([
+      'first_name' => 'Manager1', 'last_name' => 'Manager1'], 'manager1@dummysite.com'
+    );
+    $manager2 = ContactFabricator::fabricateWithEmail([
+      'first_name' => 'Manager2', 'last_name' => 'Manager2'], 'manager2@dummysite.com'
+    );
+
+    $leaveContact = ContactFabricator::fabricateWithEmail([
+      'first_name' => 'Staff1', 'last_name' => 'Staff1'], 'staffmember@dummysite.com'
+    );
+
+    $this->setLeaveApproverRelationshipTypes(['has Leaves Approved By']);
+
+    // Set manager1 and manager2 only to be leave aprovers for the leave contact
+    $this->setContactAsLeaveApproverOf($manager1, $leaveContact, null, null, true, 'has Leaves Approved By');
+    $this->setContactAsLeaveApproverOf($manager2, $leaveContact, null, null, true, 'has Leaves Approved By');
```

Comment:
You're already testing all the recipients logic on the email classes, so keep it as simple as possible here (that is, use just a single approver)